backend/app/usecases/movies/update.rb

```diff
@@ -0,0 +1,23 @@
+class Movies::Update
+  def initialize(params:, movie: Movie, repository: MoviesRepository)
+    @params = params
+    @movie = movie
+    @repository = repository
+  end
+
+  def call
```

Comment:
It can throw an error, change the name:
```suggestion
  def call!
```